docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -36,6 +36,7 @@ tasks:
     - command: git.get_project
       params:
         directory: src
+        exec_timeout_secs: 20
```

Comment:
Is this set in params or on the command (like on line 37)?

Reply:
oop yes it should be a little higher